tests/core/management/commands/test_repair_tree.py

```diff
@@ -45,6 +46,7 @@ def test_check_broken_tree_fields(load_test_data: None) -> None:
     assert "lft: 2 → 1" in err
 
 
+@pytest.mark.skip(reason="Broken tree caught by deferrable tree constraints")
 @pytest.mark.django_db
 def test_fix_broken_tree_fields(load_test_data: None) -> None:
```

Comment:
Is there a reason to keep these tests?

Reply:
Yes, they will be re-enabled in #3075. I just skipped them here while dividing this PR into three because the tests failed in this kind of intermediate step.